src/main.cpp

```diff
@@ -48,14 +48,31 @@ int main( int argc, char *argv[]) {
         }
     } else if(pipe.isOnlyInstance()) {
 
-        if(argv[1] == NULL) {
-            std::cerr << "Error track name invalid" << std::endl;
+        if(argc < 1) {
+            std::cerr << "Error, missing arguments." << std::endl;
             return -2;
         }
 
         std::string music_dir;
         std::string track_dir;
-        std::string track_name = argv[1];
+        std::string track_name;
+        std::string subdirectory;
+
+		//Check command line arguments
+        for(int i(1); i < argc; ++i)
```

Comment:
@Dyrand Not that it's a big deal but for the sake of consistency/clarity I think it should be `for(int i = 1; i < argc; ++i)`